.github/workflows/ci.yml

```diff
@@ -226,14 +229,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Release to the private PyPI repository
-        uses: pyansys/actions/release-pypi-private@v3
+        uses: pyansys/actions/release-pypi-private@v4
```

Comment:
The release actions for @v4 required the usage of the wheelhouse action. Make sure this job is present.